final_project/js/modules/Weather.js

```diff
@@ -0,0 +1,27 @@
+export class Weather {
```

Comment:
wdyt if every member becomes static? In fact, you use `Weather` as a singleton. It plays a service role and you're not going to use it to create new instances